packages/github-actions/README.md

```diff
@@ -25,11 +27,11 @@ Custom GitHub actions that help to composite GitHub workflows across the repos m
 - [`stylelint-annotation`](actions/stylelint-annotation) - Annotate stylelint results via stylelint formatter
 - [`update-version-tags`](actions/update-version-tags) - Update version tags
 
-## Prerequisites
+## Development
 
 ### JavaScript actions
 
-1. Install `node` with version >= 14
+1. Install `node` with version v20 (recommended) or v18 (compatible)
```

Comment:
Why not just go with v20? Otherwise, which specific version of v18 can I install or is compatible?

Reply:
Node.js v18 was listed because it's also compatible to use in development. Considering the End-of-Life date for v18 is within a year, it should be better to recommend v20 only. Thanks for the suggestion! Adjusted in https://github.com/woocommerce/grow/pull/123/commits/c97deef5de7472ef7e61bca22cd3a4b2a3b3e24a.